### PR TITLE
Bump artifacts dependencies

### PR DESCRIPTION
**Pull request by angelapwen**
### Merge / deployment checklist

- [ ] Confirm this change is backwards compatible with existing workflows.
- [ ] Confirm the [readme](https://github.com/github/codeql-action/blob/main/README.md) has been updated if necessary.
- [ ] Confirm the [changelog](https://github.com/github/codeql-action/blob/main/CHANGELOG.md) has been updated if necessary.

### CODE REVIEW

**Comment by angelapwen** [comment]
Okay... it looks like uploading to a different artifact directory (name) each time will work, but this will break how some of our PR checks look for debug artifacts so I'll have to update those too.

Also, I'm trying to think of any other consumers we have that might expect the debug directories to look a certain way 🤔

File: src/debug-artifacts.ts [hunk 1]
```diff
@@ -47,16 +58,29 @@ export async function uploadDebugArtifacts(
...
-      sanitizeArifactName(`${artifactName}${suffix}`),
-      toUpload.map((file) => path.normalize(file)),
-      path.normalize(rootDir),
-      {
-        continueOnError: true,
-        // ensure we don't keep the debug artifacts around for too long since they can be large.
-        retentionDays: 7,
-      },
-    );
+    if (ghVariant === GitHubVariant.GHES) {
```
Comment (aeisenberg) [inline src/debug-artifacts.ts]: Do you need to check for the GHES version here?

**Comment by angelapwen** [comment]
I split out a bunch of the refactoring into https://github.com/github/codeql-action/pull/2475 so that this PR can stick to bumping the dependencies.

File: src/debug-artifacts.ts (hunk 1 above)
Comment (angelapwen) [reply src/debug-artifacts.ts]: I don't think so — no versions of GHES support the new artifacts dependencies yet 🤦

**Comment by angelapwen** [comment]
Closing in favor of https://github.com/github/codeql-action/pull/2482.

**Pull request closed without merging**